src/mango/src/mango_cursor_text.erl

```diff
@@ -631,10 +619,11 @@ t_execute_more_results(_) ->
     ),
     meck:expect(mango_selector_text, convert, [selector], meck:val(query)),
     meck:expect(mango_selector, match, fun(selector, {doc, _}) -> true end),
-    ?assertEqual({ok, {acc, 6}}, execute(Cursor, fun foo:normal/2, {acc, 0})),
-    ?assertEqual(3, meck:num_calls(couch_stats, increment_counter, 1)),
-    ?assertEqual(3, meck:num_calls(mango_execution_stats, incr_docs_examined, 1)),
-    ?assertEqual(3, meck:num_calls(mango_execution_stats, incr_results_returned, 1)).
+    meck:expect(chttpd_stats, incr_rows, [0], meck:val(ok)),
```

Comment:
@iilyak, @chewbranca please note that `text` search will not modify the `rows` value.  Is this acceptable or shall they also update the value?  If yes, how?

Reply:
I have talked about this with @willholley and @mikerhodes offline.  The conclusion was that `rows` should be equal to `reads` for `text` indexes.  The distinction between these two categories make difference only for `json` indexes, especially when covering indexes are in use.